### PR TITLE
Added nblink option to control placement of notebook links

### PR DESCRIPTION
**Pull request by jlstevens**
Should be a fairly simple change that also introduces some tests that check the generated rst content.

### CODE REVIEW

File: nbsite/__main__.py [hunk 1]
```diff
@@ -32,6 +32,7 @@ def main(args=None):
     generaterst_parser.add_argument('--branch',type=str,help='where to init doc',default='master')
     generaterst_parser.add_argument('--offset',type=int,help='where to init doc',default=0)
     generaterst_parser.add_argument('--overwrite',type=int,help='where to init doc',default=False)
+    generaterst_parser.add_argument('--nblink',type=int,help='where to to place notebook links',default='bottom')
```
Comment (jbednar) [inline nbsite/__main__.py]:
```suggestion
    generaterst_parser.add_argument('--nblink',type=str,help='where to to place notebook links',default='bottom')
```

File: nbsite/__main__.py [hunk 2]
```diff
@@ -32,6 +32,7 @@ def main(args=None):
     generaterst_parser.add_argument('--branch',type=str,help='where to init doc',default='master')
     generaterst_parser.add_argument('--offset',type=int,help='where to init doc',default=0)
     generaterst_parser.add_argument('--overwrite',type=int,help='where to init doc',default=False)
+    generaterst_parser.add_argument('--nblink',type=str,help='where to to place notebook links',default='bottom')
```
Comment (jsignell) [inline nbsite/__main__.py]: Can there be a list of valid options?

File: nbsite/tests/test_cmd.py [hunk 3]
```diff
@@ -162,6 +162,55 @@ def test_generate_rst_with_skip_one_notebook(tmp_project):
...
+def test_generate_rst_without_nblink(tmp_project):
+
+    expected = ['******************',
+                'Example Notebook 1',
+                '******************',
+                '', '.. notebook:: test_project ../examples/Example_Notebook_1.ipynb',
+                '    :offset: 0']
+    project = tmp_project
+    generate_rst("test_project", project_root=str(project),
+                 nblink='none', skip='.*Notebook_0.*')
```
Comment (jsignell) [inline nbsite/tests/test_cmd.py]: just don't include `nblink`

**Comment by jbednar** [comment]
Looks good!

**Comment by jsignell** [comment]
Yep I will merge this and tag when green unless anyone has qualms

**Pull request merged**